### ISSUE TITLE
An error

### ISSUE THREAD

**Issue by Mirza5138** [arcusmaximus/YTSubConverter#36]
Hello, im the guy that commented recently. So i made a video to test it but there is a problem and i don't know why. Here is the video, check subtitles. https://www.youtube.com/watch?v=KbM2q1fc6Oc

Also im having an error when i try to close app. I'm using Turkish so i don't know if im true but i think it says "Can not find path C:\WINDOWS\system32\<file name i forgot>.xml"

**Comment by arcusmaximus** [comment]
What exactly is the problem in the test video? Do the subtitles look different than in Aegisub?

And by "error when you close the app", do you mean Aegisub or YTSubConverter? Could you check again to see what file it's complaining about?

**Comment by Mirza5138** [comment]
Check the video subtitles. Also i meant while closing YTSubConverter.

**Comment by Mirza5138** [comment]
> What exactly is the problem in the test video? Do the subtitles look different than in Aegisub?
> 
> And by "error when you close the app", do you mean Aegisub or YTSubConverter? Could you check again to see what file it's complaining about?

It is StyleOptions.xml

**Comment by arcusmaximus** [comment]
The subtitles look like this for me - I don't immediately see what's wrong. Do they look different for you?
![image](https://user-images.githubusercontent.com/133129/114669593-5b2b6c80-9d02-11eb-817f-1a3fe7840b2e.png)

As for StyleOptions.xml: this is a file that YTSubConverter creates "next to" itself (in the same folder as itself). I'm not sure why it would try to put it in the system32 folder. Did you place YTSubConverter in system32 by any chance?

**Comment by Mirza5138** [comment]
I didnt. I just downloaded it. Also subtitles looks different for me.

**Comment by Mirza5138** [comment]
Wait subtitles are fixed now! StyleOptions.xml is not that important for now i think.

**Comment by arcusmaximus** [comment]
Hah, alright. Closing this issue then.

**Comment by louie-github** [comment]
Hi! I'm not sure whether to open a new issue or not, but I'm getting the same error described as above:

When closing YTSubConverter, this error appears:
```
System.UnauthorizedAccessException: Access to the path 'C:\WINDOWS\system32\StyleOptions.xml' is denied.
```

I've noticed that it only happens when starting YTSubConverter from the Start Menu (via Search). It does not happen when clicking on the executable to start it.

**Comment by arcusmaximus** [comment]
Aha, then I know what the problem is. Should be fixed now - please try the new 1.5.3 release. Thanks for the information!
